src/renderer/theme.store.ts

```diff
@@ -46,7 +46,6 @@ export interface ThemeItems extends Theme {
 }
 
 export class ThemeStore extends Singleton {
-  static readonly defaultTheme = "lens-dark";
```

Comment:
Is this a breaking change? It could be kept, assigned to `defaultTheme` from vars?

Reply:
It is only a breaking change if we export `ThemeStore`, which do don't we only export `src/extensions/renderer-api/theming.ts`